frontend/src/features/admin-form/common/components/PreviewFormBanner/PreviewFormBanner.tsx

```diff
@@ -166,16 +166,32 @@ export const PreviewFormBanner = ({
         </Drawer>
       </Flex>
       {isPaymentEnabled && (
-        <Flex backgroundColor={'#3A3E46'}>
-          <Text
-            textStyle="body-2"
-            color="white"
-            ml="2rem"
-            mt="0.5rem"
-            mb="0.5rem"
-          >
-            Payments made in Form Preview mode will not reflect on Stripe.
-          </Text>
+        <Flex backgroundColor="neutral.900">
+          {process.env.SECRET_ENV === 'production' ? (
+            <Text
+              textStyle="body-2"
+              color="white"
+              ml="2rem"
+              mt="0.5rem"
+              mb="0.5rem"
+            >
+              To test your payment form, replicate this form on our{' '}
+              <Link isExternal color="white" href="https://uat.form.gov.sg">
```

Comment:
Shall we move the uat link out to `~constants/links`? 

As we gon use uat more, perhaps it will be good to just keep it in consts

Reply:
I did consider this, but wasn't really sure if we'd use the link in other instances? 🤔 maybe we can move it when we actually use it?

Reply:
nit: feel like it'd be better to have it in `~constants/links` for consistency, some of the links there are also only used once.

Reply:
okays i'll do this then!

Reply:
thanks friends, done in [2276aa5](https://github.com/opengovsg/FormSG/pull/6327/commits/2276aa540b72ab54cca9bebc432a0b16ff5e23fc)